db-scripts/src/main/resources/cgds.sql

```diff
@@ -97,9 +97,8 @@ DROP TABLE IF EXISTS `type_of_cancer`;
 DROP TABLE IF EXISTS `geneset_hierarchy_leaf`;
 DROP TABLE IF EXISTS `geneset_hierarchy_node`;
 DROP TABLE IF EXISTS `geneset`;
-DROP TABLE IF EXISTS `genetic_entity`;
-DROP TABLE IF EXISTS `reference_genome_gene`;
```

Comment:
It is true .. this is a small bugfix that helped with our unit testing. When using an actual persistent external test database, the tables must be dropped in the correct order before setting them up for the next run (the foreign key constraints must be respected with innodb). This fix  could be a separate PR, but since it does not alter the schema and does not seem to affect many others it seemed ok to just fix it as part of this work.